### PR TITLE
Add x86_64-darwin target for libvpx

### PR DESCRIPTION
**Pull request by uvjustin**
CIBW builds the x86_64 MacOS X wheels to the minimum target of "10.9" (Darwin 13): https://cibuildwheel.readthedocs.io/en/stable/cpp_standards/#macos-and-deployment-target-versions
To match this, it looks like we need to add this target to the configure args when building libvpx for x86_64 macs.

### CODE REVIEW

**Comment by uvjustin** [comment]
It just looked like one of the downloads failed causing the CI to fail. Stage 2 and 3 of ppc64le take like 16 hours on these runners...the CI finished OK on my fork: https://github.com/uvjustin/pyav-ffmpeg/actions/runs/2289807338

File: scripts/cibuildpkg.py [hunk 1]
```diff
@@ -222,8 +222,11 @@ def _build_with_autoconf(self, package: Package, for_builder: bool) -> None:
                     "--build=x86_64-apple-darwin",
                     "--host=aarch64-apple-darwin",
                 ]
-        elif platform.system() == "Windows" and package.name == "vpx":
-            configure_args += ["--target=x86_64-win64-gcc"]
+        elif package.name == "vpx":
+            if platform.system() == "Windows":
+                configure_args += ["--target=x86_64-win64-gcc"]
+            if platform.system() == "Darwin":
+                configure_args += ["--target=x86_64-darwin13-gcc"]
```
Comment (uvjustin) [inline scripts/cibuildpkg.py:229]: Should we put a note in here about darwin13 matching the cibuildwheel minimum macos version?

Comment (jlaine) [reply scripts/cibuildpkg.py:229]: Yes any notes are welcome for our future selves :)

**Comment by jlaine** [comment]
I really hate these packages whose authors think they are so smart it's worth having their own custom build system..

**Comment by jlaine** [comment]
I've tried to refactor your changes to group all vpx-specific arguments, let's see if the build passes.

**Comment by uvjustin** [comment]
Probably need to restart the checks as the windows build failed due to a download error

**Comment by jlaine** [comment]
> Probably need to restart the checks as the windows build failed due to a download error

I have no idea why the xz download is so flaky, but it's a real nuisance.. I've re-triggered the build.

**Pull request merged**